lib/ruby_lsp/ruby_lsp_rails/server.rb

```diff
@@ -332,6 +332,17 @@ def execute(request, params)
           with_request_error_handling(request) do
             send_result(resolve_route_info(params))
           end
+        when "i18n"
+          with_request_error_handling(request) do
+            result = resolve_i18n_key(params.fetch(:key))
+            send_result(result)
+          end
+        when "reload_i18n"
+          with_progress("rails-reload-i18n", "Reloading Ruby LSP Rails I18n") do
+            with_notification_error_handling(request) do
+              I18n.reload! if defined?(I18n) && I18n.respond_to?(:reload!)
```

Comment:
This is provided because other I18n backend maybe does not implement the reloading feature.